src/components/MyProfile.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import ListReservedMissions from './missions/ListReservedMissions';
+import ReservedRocket from './rockets/reservedRocket';
+
+const MyProfile = () => {
+  const reservedRockets = (rockets) => rockets.filter((rocket) => rocket.reserved === true);
+  const rockets = useSelector((state) => reservedRockets(state.rockets.rockets));
+
+  return (
+    <div className="profilesCont">
+      <div className="missionsCont">
+        <h3 className="heading">My Missions</h3>
+        <ListReservedMissions />
+      </div>
+      <div className="rocketsCont">
+        <h3 className="heading">My Rockets</h3>
+
+        {rockets.map((rocket) => (
+          <ReservedRocket key={rocket.id} rocket={rocket} />
+        ))}
+        { rockets.length === 0 && <p className="firstJoin">&emsp; No Reserved Rocket </p>}
+      </div>
+    </div>
```

Comment:
- [ ] Please 🙏🏽 use the underlined style in the MyProfile link like you did with the others to be similar to the original design.
- [ ] Kindly use two columns to list the missions and rockets for a better user experience.

| Original design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/209418306-6ec9eb91-4991-43ae-8160-1fd40db7866e.png) | ![image](https://user-images.githubusercontent.com/58642949/209418378-e8107866-4fcd-44da-b05d-e58f0f9fcc09.png) |

Reply:
This change has been done well